rmf_robot_sim_common/src/slotcar_common.cpp

```diff
@@ -538,6 +673,113 @@ std::pair<double, double> SlotcarCommon::update(const Eigen::Isometry3d& pose,
   return velocities;
 }
 
+std::pair<double, double> SlotcarCommon::update_nonholonomic(
```

Comment:
Same as the callback I think it would be useful to maybe put a short comment before chunks of code to explain what they do, or break it down a bit.

Reply:
add comments 1ec3bde